sim_pipeline/Plots/gg_plots.py

```diff
@@ -2,7 +2,6 @@
 from astropy.visualization import make_lupton_rgb
 from sim_pipeline.image_simulation import simulate_image
 
```

Comment:
PEP8 standards require two white spaces before a class definition. If you are using PyCharm, you can have a plugin that highlights such things